### PR TITLE
Add client, r?ferjm

### PR DESCRIPTION
**Pull request by michielbdejong**
This PR adds the client code from https://github.com/fxbox/foxbox/pull/163, which is a first step towards making this a stand-alone component which can be used independently from foxbox.

r? @ferjm

### CODE REVIEW

File: client/proxy.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+var fs = require('fs');
+var exec = require('child_process').exec;
+var proxy = require('http-proxy').createProxyServer();
+var ports = {
+  backend: 8000,
+  publicLocal: 4334
+};
+var os = require('os');
+var TUNNEL_IPADDR = '52.36.71.23';
+var DNS_ZONE = 'box.knilxof.org';
```
Comment (ferjm) [inline client/proxy.js:14]: Shouldn't we get this from command line?

**Comment by ferjm** [comment]
LGTM. Thanks

File: client/proxy.js (hunk 1 above)
Comment (michielbdejong) [inline client/proxy.js:14]: Yes, this is done in the follow-up, see #18.

**Comment by michielbdejong** [comment]
Thanks!

**Pull request merged**